src/MarkdownParser.cc

```diff
@@ -13,7 +13,7 @@
 using namespace mdp;
 
 const size_t MarkdownParser::OutputUnitSize = 64;
-const size_t MarkdownParser::MaxNesting = 32;
+const size_t MarkdownParser::MaxNesting = 128;
```

Comment:
Not entirely sure what to do in regards to adding a test for this. Before we added a specific test regarding ["parsing six nested level list items"](https://github.com/apiaryio/markdown-parser/pull/13/files#diff-ac1344b6c620735354458b48fe6f2271R577). Should I take that test and increase it to 100 or so items?